doc/main.html

```diff
@@ -1235,6 +1249,272 @@ <h3>Git commit hooks</h3>
+            <td><code>DP</code></td>
+            <td><code>pub</code></td>
+          </tr>
+          <tr>
+            <td>ST audit complete (or) waiting for publication</td>
+            <td><code>PUB</code></td>
+            <td><code>draft</code></td>
+          </tr>
+          <tr>
+            <td>Published</td>
+            <td><code>PUB</code></td>
+            <td><code>pub</code></td>
+          </tr>
+        </tbody>
+      </table>
+
+      <p>
+        See <a href="#sec-sample-workflow"></a> for a sample workflow utilizing these statuses. 
+      </p>
+
+    </section>
+
+    <section id="sec-document-github-releases">
+      <h3>GitHub Releases</h3>
+      <p>
+        Whenever the status of the document dictates as per <a href="#tab-document-mapping-status"></a>, that a merged commit to the main branch that updates the document to <code>&lt;meta itemprop="pubState" content="pub" /&gt;</code>, a release shall be created in the GitHub repo. 
+      </p>
+      <p>
+        The GitHub release shall be named <code>rYYYY-pubState</code>, with a tag assigned to the release that matches the release name. If creating a CD, itterate the <code>pubState</code>, starting with <code>1</code>. 
+      </p>
+      <p class="example">
+        <code>r2023-fcd</code> or <code>r2023-cd2</code>
+      </p>
+      <p class="note">
+        The itteration of CD is to account for multiple possible PCD periods. 
+      </p>
+      <p>
+        When creating the next update on the repo, update <code>latestEditionTag</code> in the <code>.smpte-build.json</code> file to the value equal to the tag name of this latest release. This will trigger the tooling to create a link for <code>Redline to most recent edition</code> during any PRs and merges to the main branch. 
+      </p>
+
+      <p>
+        See <a href="#sec-sample-workflow"></a> for a sample workflow utilizing releases. 
+      </p>
+
+    </section>
+
+    <section id="sec-sample-workflow">
+      <h3>Sample Workflow </h3>
+      <p>
+        <a href="#tab-sample-workflow"></a> shows a sample workflow would be the steps for an Engineering Document going through the an initial draft or revision and the balloting process. 
+      </p>
+      <table id="tab-sample-workflow">
+        <caption>Sample workflow</caption>
+        <thead>
+          <tr>
+            <th>Workflow Step</th>
+            <th>Action(s) on document and/or GitHub repo</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>(1) - Project is approved</td>
+            <td>
+              <ol>
+                <li>If document is an intial draft, skip to Workflow Step (2)</li>
+                <li>If document is a revision, skip to Workflow Step (3)</li>
+              </ol>
+            </td>
+          </tr>
+          <tr>
+            <td>(2) - Project is a document initial draft</td>
+            <td>
+              <ol>
```

Comment:
```suggestion
              <ol type="a">
```

Reply:
Addressed in https://github.com/SMPTE/html-pub/pull/139/commits/6ab6d4dd994fa0751b01ac792613709907ee3b96